pandas/core/common.py

```diff
@@ -454,3 +454,21 @@ def _pipe(obj, func, *args, **kwargs):
         return func(*args, **kwargs)
     else:
         return func(obj, *args, **kwargs)
+
+
+def get_rename_function(mapper):
```

Comment:
can you make this private (e.g. _get_rename_function)

Reply:
done